xbstrap/base.py

```diff
@@ -2614,6 +2615,33 @@ def run_tool_task(cfg, task):
     )
 
 
+def discard_src(cfg, src):
+    with suppress(Exception):
+        shutil.rmtree(os.path.join(cfg.source_root, src.source_subdir))
+
+
+def discard_pkg(cfg, pkg):
+    with suppress(Exception):
```

Comment:
why?

Reply:
Looks like this was done to make the code not error if the folder doesn't exist